resources/definitions/fdmprinter.def.json

```diff
@@ -1053,6 +1053,55 @@
                         }
                     }
                 },
+                "beading_strategy_type":
+                {
+                    "label": "Beading Strategy",
+                    "description": "Beading strategy used by libArachne to generate walls.",
```

Comment:
Is the lemma "used by libArachne" of any insterest to the user?

It would be nice if the strategies were clarified here instead.

Reply:
It is still a work in progress, so we haven't settled down to the description that will be used in the end. I left the descriptions like that for now so that the settings can be easily found while we still integrate libArachne, by searching 'libArachne' or 'beading'.

Reply:
Seems you forgot to go over it later on...